robottelo/hosts.py

```diff
@@ -703,6 +704,8 @@ def register(
             options['ignore-subman-errors'] = str(ignore_subman_errors).lower()
         if force:
             options['force'] = str(force).lower()
+        if download_utility is not None:
+            options['download-utility'] = download_utility
```

Comment:
@dosas Do you intent to add some test to cover this option?

Reply:
Obviously I did not intent to ;) but if you point me in the right direction I can add a test.